timemachine/fe/free_energy.py

```diff
@@ -591,7 +596,10 @@ def run_sims_with_greedy_bisection(
     temperature: float
         Temperature in K
 
-    verbose: bool
+    min_overlap: float or None, optional
+        If not None, return early when the BAR overlap between all neighboring pairs of states exceeds this value
```

Comment:
Clarification: Is the threshold defined in terms of BAR overlap or [2x](https://github.com/proteneer/timemachine/blob/deccde794e82079ca55ced20baa85f1a42f3b725/timemachine/fe/bar.py#L265) BAR overlap? (i.e. normalized to [0, 0.5] or [0, 1]?)

Reply:
This uses the definition here: https://github.com/proteneer/timemachine/blob/deccde794e82079ca55ced20baa85f1a42f3b725/timemachine/fe/bar.py#L265

My understanding is that this is the standard definition of "scalar overlap". We might consider updating this line to

```python
return pymbar.MBAR(u_kn, N_k).computeOverlap()["scalar"]
```

to minimize future confusion?

Reply:
Ahh, sorry, I was confused. Correct as-is!

"Scalar overlap" sounds more self-documenting